app/src/main/java/com/example/playlistmaker/AudioPlayerActivity.kt

```diff
@@ -22,6 +28,22 @@ class AudioPlayerActivity : AppCompatActivity() {
     private lateinit var genre: TextView
     private lateinit var country:TextView
     private lateinit var songTime: TextView
+    private lateinit var url: String
+    private lateinit var play: ImageView
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer by lazy { MediaPlayer() }
```
Таким образом, плеер будет инициализирован только при первом доступе к переменной (при создании активити не будут сразу тратится ресурсы на создание плеера). Плюс лучше делать `val` и пользоваться одним и тем же объектом плеера, не пересоздавая его, так как апи плеера позволяет так делать